Refactor documentation (#364)
scalar-labs/scalardb: README.md

[![CircleCI](https://circleci.com/gh/scalar-labs/scalardb/tree/master.svg?style=svg&circle-token=672f70ce7f2c4f8d9e71f7c9db8ae824e2cfaeca)](https://circleci.com/gh/scalar-labs/scalardb/tree/master)

> The major features of Scalar DB are as follows:

Reviewer: ```suggestion
Scalar DB is a universal transaction manager that achieves:
```

> - Provide a universal transaction manager that achieves database/storage-agnostic ACID transactions in a scalable manner even if an underlying database or storage is not ACID-compliant.

Reviewer: ```suggestion
- database/storage-agnostic ACID transactions in a scalable manner even if an underlying database or storage is not ACID-compliant.
```

> - Provide a two-phase commit based transaction manager that achieves ACID transactions spanning multiple (possibly different) databases/storages and services.

Reviewer: ```suggestion
- multi-storage/database/service ACID transactions that can span multiple (possibly different) databases, storages, and services.
```
